.github/workflows/build.yml

```diff
@@ -47,6 +47,13 @@ jobs:
 
     steps:
       # Checkout push-to-registry action GitHub repository
+      - name: Maximize build space
+        uses: ublue-os/remove-unwanted-software@v7
+        with:
+          remove-android: 'true'
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Why have we added an old Checkout version here? 
We'll start getting warnings about Node 14.x